app/workloadSummaryReport/services/workloadSummaryActions.js

```diff
@@ -402,6 +420,145 @@ class WorkloadSummaryActions {
+						var sectionGroup = teachingAssignment.sectionGroupId > 0 ? sectionGroups.list[teachingAssignment.sectionGroupId] : null;
+						var course = sectionGroup ? courses.list[sectionGroup.courseId] : null;
+
+						assignment.term = TermService.getTermName(termCode);
+						assignment.termCode = termCode;
+
+						assignment.description = TeachingAssignmentService.getDescription(teachingAssignment, course);
+
+						if (teachingAssignment.sectionGroupId > 0) {
+							assignment.sequencePattern = course.sequencePattern;
+							assignment.enrollment = _self._getEnrollment(sectionGroup);
+
+							assignment.actualEnrollment = sectionGroup.actualEnrollment;
+							assignment.maxEnrollment = sectionGroup.maxEnrollment;
+
+							var seats = 0;
+
+							var sections = WorkloadSummaryReducers._state.sections.bySectionGroupId[sectionGroup.id];
+
+							if (sections) {
+								sections.forEach(function (section) {
+									seats += section.seats;
+								});
+							}
+
+							assignment.seats = seats;
+							assignment.previousEnrollment = sectionGroup.previousEnrollment;
+							assignment.enrollmentPercentage = assignment.maxEnrollment && assignment.actualEnrollment ? parseInt((assignment.actualEnrollment / assignment.maxEnrollment) * 100) : "0";
+							assignment.units = _self._getUnits(course);
+							assignment.studentCreditHours = assignment.seats * assignment.units;
+
+							calculatedView.genericInstructorTotals.assignmentCount += 1;
+							calculatedView.genericInstructorTotals.seats += assignment.seats;
+							calculatedView.genericInstructorTotals.enrollment += assignment.actualEnrollment;
+							calculatedView.genericInstructorTotals.previousEnrollment += assignment.previousEnrollment;
+							calculatedView.genericInstructorTotals.units += assignment.units;
+							calculatedView.genericInstructorTotals.studentCreditHours += assignment.studentCreditHours;
+							calculatedView.genericInstructorTotals.instructorCount += 1;
+						}
+
+						genericInstructor.assignments.push(assignment);
+
+						genericInstructor.totals.units += assignment.units || 0;
+						genericInstructor.totals.studentCreditHours += assignment.studentCreditHours || 0;
+						genericInstructor.totals.enrollment += assignment.enrollment || 0;
+						genericInstructor.totals.seats += assignment.seats || 0;
+						genericInstructor.totals.actualEnrollment += assignment.actualEnrollment || 0;
+						genericInstructor.totals.previousEnrollment += assignment.previousEnrollment || 0;
+						genericInstructor.totals.assignmentCount += 1;
+					});
+
+					genericInstructor.assignments = _array_sortByProperty(genericInstructor.assignments, ["termCode", "description"]);
+
+					calculatedView.byInstructorType[genericInstructorTypeId].push(genericInstructor);
+					calculatedView.totals.instructorCount += 1;
+				});
+
+				calculatedView.unassignedCourses = unassignedCourses;
+
+				calculatedView.workloadTotals = {
```

Comment:
@ltwheeler What do you think of having object keys that are 1) capitalized and 2) have spaces?

Reply:
Hmmm,
I think we would be less likely to have typos, and their intent would be clearer to me if they were formatted like:
ASSIGNED
UNASSIGNED
TBD_INSTRUCTOR

Ideally, I would want an object like 
var workloadTotalCategories = {
ASSIGNED: "ASSIGNED",
UNASSIGNED: "UNASSIGNED",
TBD_INSTRUCTOR: "TBD_INSTRUCTOR"
}

and then we could just reference that object here during assignment:
calculatedView.workloadTotals = {
workloadTotalCategories.ASSIGNED: calculatedView.totals,
workloadTotalCategories.UNASSIGNED: calculatedView.unassignedTotals,
workloadTotalCategories.TBD_INSTRUCTOR: calculatedView.genericInstructorTotals
}

and then do the same in the template when we pull out those values for display.

It might be better to define them as a constant() in the workloadSummaryApp.js file, like we did with 'ActionTypes', as these will be used in numerous places in the app.

Reply:
Rather than using the display string as the object key, I opted to add a new property for the displayName since that's what we were using it for. I also moved the calculations from the template to `workloadSummaryActions.js`